docs/tans.rst

```diff
@@ -170,3 +170,49 @@ Reference
    :inherited-members:
    :member-order: bysource
    :exclude-members: is_unset, naive_parse, print_nested
+
+
+.. _tans-full-example:
+
+Full example
+------------
+
+A full example on how to get transactions if a TAN is required. If a TAN is required `result` will be an object of type `NeedTANResponse`. Otherwise it will hold your transactions directly.
```

Comment:
reStructuredText usually uses double backtick for code
```suggestion
A full example on how to get transactions if a TAN is required. If a TAN is required ``result`` will be an object of type ``NeedTANResponse``. Otherwise it will hold your transactions directly.
```